### DISCUSSION TITLE
flutter_inappwebview 6.1.0 with InAppWebView widget macOS and Windows support

### DISCUSSION THREAD

**Discussion by pichillilorenzo** [ente-io/ente#3494] [General]
Hello ente-io and @mnvr, I'm the author of the [flutter_inappwebview](https://github.com/pichillilorenzo/flutter_inappwebview/) Flutter plugin and I found out that your project is using it!
I recently updated the plugin with InAppWebView widget macOS and Windows support!

I would really like to add your project to the [Showcase](https://inappwebview.dev/showcase) page of the official plugin website.
It shows an open list of apps built with Flutter InAppWebView (both open and closed source).

So, may I have your permission to add it there?
If yes, I would like to have just a simple description of what `flutter_inappwebview` plugin was used for, and for which feature in your app.
For the screenshots and app logo, maybe, I could use the ones you have in your repository or on the app stores.

Hope to receive a positive feedback, thanks a lot!

**Comment by vishnukvmd** [comment]
Yes, please feel free to add Ente to your show case. Thank you for creating and maintaining flutter_inappwebview!

**Reply by pichillilorenzo** [reply]
Apps added! Thanks a lot again!
